src/main/webapp/WEB-INF/views/developerTable.jsp

```diff
@@ -2,31 +2,34 @@
 <%@ page import="org.example.model.Developer" %>
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <title>Developers Table</title>
+  <meta charset="UTF-8">
+  <title>Developers Table</title>
 </head>
-    <body>
-        <center>
-            <table>
-                <h2>Developers Table</h2>
-                    <tr>
-                        <th><%= Developer.COLUMN_NAME_ID %></th>
-                        <th><%= Developer.COLUMN_NAME_FIRST_NAME %></th>
-                        <th><%= Developer.COLUMN_NAME_SECOND_NAME %></th>
-                        <th><%= Developer.COLUMN_NAME_AGE %></th>
-                        <th><%= Developer.COLUMN_NAME_PROGRAMMING_LANG %></th>
-                    </tr>
-                     <c:forEach var="developer" items="${developerList}">
-                    <tr>
-                        <td>${developer.id}</td>
-                        <td>${developer.firstName}</td>
-                        <td>${developer.secondName}</td>
-                        <td>${developer.age}</td>
-                        <td>${developer.progLang}</td>
-                    </tr>
-                </c:forEach>
-            </table>
-        </center>
-    </body>
-</html>
+
+<body>
+  <center>
+    <table>
+      <h2>Developers Table</h2>
+      <tr>
+        <th><%= Developer.COLUMN_NAME_ID %></th>
+        <th><%= Developer.COLUMN_NAME_FIRST_NAME %></th>
+        <th><%= Developer.COLUMN_NAME_SECOND_NAME %></th>
+        <th><%= Developer.COLUMN_NAME_AGE %></th>
+        <th><%= Developer.COLUMN_NAME_PROGRAMMING_LANG %></th>
+      </tr>
+      <c:forEach var="developer" items="${developerList}">
+        <tr>
+          <td>${developer.id}</td>
+          <td>${developer.firstName}</td>
+          <td>${developer.secondName}</td>
+          <td>${developer.age}</td>
+          <td>${developer.progLang}</td>
+        </tr>
+      </c:forEach>
+    </table>
+  </center>
+</body>
+
+</html>
```

Comment:
Always add new line at the end of the file

Reply:
fixed